pyglmnet/pyglmnet.py

```diff
@@ -232,7 +169,7 @@ def copy(self):
         """Return a copy of the object."""
         return deepcopy(self)
 
-    def qu(self, z):
+    def __qu(self, z):
```

Comment:
woops, I wish I had finished that book ;) python has so many features I'm learning all the time. Anyway, I found this on stack overflow: http://stackoverflow.com/a/6930223. Seems like there is a good reason everyone uses a single underscore